x-pack/legacy/plugins/siem/server/lib/detection_engine/rules/prepackaged_rules/windows_uac_bypass_event_viewer.json

```diff
@@ -1,36 +1,35 @@
-{
-  "description": "Identifies User Account Control (UAC) bypass via eventvwr.exe. Attackers bypass UAC to stealthily execute code with elevated permissions.",
-  "index": [
-    "winlogbeat-*"
-  ],
-  "language": "kuery",
-  "max_signals": 33,
```

Comment:
Shouldn't this result in a version bump?

Reply:
we will re-check those in the next build.